hercules/python_simulators/solar_pysam.py

```diff
@@ -61,7 +61,7 @@ def step(self, inputs):
         for k, v in self.model_params.items():
             try:
                 system_model.value(k, v)
-            except KeyError:
+            except Exception:
```

Comment:
This was fixed in my last PR to `develop` so I'm applying the same change here to prevent errors.